### PR TITLE
Add common library service

### LINKED ISSUE #79: Add common library service
Connect backend to Common Library API

### PR DESCRIPTION
**Pull request by GodVenn**
Closes #79

### CODE REVIEW

**Comment by GodVenn** [comment]
The tests failing are due to NuGet not finding the Equinor package source. Will fix.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,17 @@ dotnet run
 ```
 The default URL is set to localhost:5000.
 
+To access the /Country endpoint you need to have the client secret of our app from azure available. 
+One way to do this for now is to add a ``appsettings.Development.json`` file, which you can make from the ``appsettings.Development.json.example`` file in which you can store the ClientSecret in the following format.
```
Comment (maiaht) [inline README.md:32]: In markdown files the line length should not exceed 80 characters

File: README.md [hunk 2]
```diff
@@ -28,6 +28,17 @@ dotnet run
...
+To access the /Country endpoint you need to have the client secret of our app from azure available. 
+One way to do this for now is to add a ``appsettings.Development.json`` file, which you can make from the ``appsettings.Development.json.example`` file in which you can store the ClientSecret in the following format. 
+It is important that the file is named as suggested, to guarantee that it will be ignored by git, since it will now contain a secret.
+```
+    "AzureAd": {
+        ...
+        "ClientSecret": "XXXXXXXXXXX",
+        ...
+    },
+```
```
Comment (maiaht) [inline README.md:40]: Is it necessary to include these lines?

File: backend/api/Services/CommonLibraryService.cs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        public async Task<List<string>> GetAllCountries()
+        {
+            _logger.LogInformation("Attempting to retrieve country list from common library");
+
+            List<string> countries = await _commonLibraryClient.CodeNameListAsync("Country", isValid:true);
+
+            _logger.LogInformation("Successfully retrieved country list from common library");
+
+            return countries;
+        }
```
Comment (maiaht) [inline backend/api/Services/CommonLibraryService.cs:40]: Maybe there should be a try-catch statement in this function?

**Review by maiaht — APPROVED** [review]

**Pull request merged**